perfkitbenchmarker/linux_benchmarks/bonnie_benchmark.py

```diff
@@ -223,10 +277,23 @@ def ParseCSVResults(results):
         If a 4th element is included, it is a dictionary of sample
         metadata.
   """
+  results = results.split(',')
+
+  if results[0] == "1.98":
```

Comment:
suggest make a  dictionary. e.g.
supported_parser = {'1.96': BONNIE_RESULTS_MAPPING_1_96, '1.97': ..._1_97, ..}

Such that you can simply here, rather than copy-pasting the log lines 
version = results[0]
if version in supported_parser:
  logging...
  BONNIE_RESULTS_MAPPING = supported_parser[version]
else:
  raise ...

Reply:
That's a better approach, done.